### PR TITLE
Added RxAudioPlayer.progress

### PR DESCRIPTION
**Pull request by wieczorek1990**

### CODE REVIEW

File: rxandroidaudio/src/main/java/com/github/piasy/rxandroidaudio/RxAudioPlayer.java [hunk 1]
```diff
@@ -47,6 +47,13 @@
 
     MediaPlayer mPlayer;
 
+    public int progress() {
```
Comment (Piasy) [inline rxandroidaudio/src/main/java/com/github/piasy/rxandroidaudio/RxAudioPlayer.java]: Why put this method on top of the file? And could we have a rx version?

**Comment by wieczorek1990** [comment]
Rearranged code with formatter.
I don't need the rx version.

**Comment by Piasy** [comment]
Well, thanks for your pr.

**Comment by wieczorek1990** [comment]
Thank you. Waiting for release so I can pull your library from jcenter instead of jar file.

**Comment by Piasy** [comment]
`1.2.0` is now available.

**Pull request merged**